### PR TITLE
[MInor-Fix][366] Add textfield to the "Small Banner”.

### PR DESCRIPTION
**Pull request by ivan-berezhnov**
Issue #366 
Drupal.org issue: https://www.drupal.org/node/2882621

Steps:
- [x] log in as admin
- [x] go to frontpage or any page
- [x] click 'Edit' above main content
- [x] add small banner
- [x] add some text to the description for small banner also headline text and image.
- [x] save
- [x] verify you can see your text in small banner also check if description empty you don't see in page source div with class "banner-description".

### CODE REVIEW

**Comment by gundevel** [comment]
Build comment file: 
--------------
===============
 
| Environment | Link |
| ------------- | ------------- |
| Fresh OpenY installation | http://ci.openymca.org/build1252 |
| Upgraded(upgrade path) installation | http://upgrade.openy-dev.ffwua.com/build1252 |
| Installation process | http://install.openy-dev.ffwua.com/build1252/install.php |

--------------
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.openymca.org:8080/job/PR_BUILDER/1252/

**Comment by gundevel** [comment]
Behat test results file URL: http://ci.openymca.org/build1252/profiles/contrib/openy/build/reports/behat

**Comment by gundevel** [comment]
Build comment file: 
--------------
===============
 
| Environment | Link |
| ------------- | ------------- |
| Fresh OpenY installation | http://ci.openymca.org/build1311 |
| Upgraded(upgrade path) installation | http://upgrade.openy-dev.ffwua.com/build1311 |
| Installation process | http://install.openy-dev.ffwua.com/build1311/install.php |

--------------
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.openymca.org:8080/job/PR_BUILDER/1311/

**Comment by gundevel** [comment]
Behat test results file URL: http://ci.openymca.org/build1311/profiles/contrib/openy/build/reports/behat

**Comment by gundevel** [comment]
Build comment file: 
--------------
===============
 
| Environment | Link |
| ------------- | ------------- |
| Fresh OpenY installation | http://ci.openymca.org/build1314 |
| Upgraded(upgrade path) installation | http://upgrade.openy-dev.ffwua.com/build1314 |
| Installation process | http://install.openy-dev.ffwua.com/build1314/install.php |

--------------
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.openymca.org:8080/job/PR_BUILDER/1314/

**Comment by gundevel** [comment]
Behat test results file URL: http://ci.openymca.org/build1314/profiles/contrib/openy/build/reports/behat

**Comment by gundevel** [comment]
Build comment file: 
--------------
===============
 
| Environment | Link |
| ------------- | ------------- |
| Fresh OpenY installation | http://ci.openymca.org/build1315 |
| Upgraded(upgrade path) installation | http://upgrade.openy-dev.ffwua.com/build1315 |
| Installation process | http://install.openy-dev.ffwua.com/build1315/install.php |

--------------
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.openymca.org:8080/job/PR_BUILDER/1315/

**Comment by gundevel** [comment]
Behat test results file URL: http://ci.openymca.org/build1315/profiles/contrib/openy/build/reports/behat

**Comment by gundevel** [comment]
Build comment file: 
--------------
===============
 
| Environment | Link |
| ------------- | ------------- |
| Fresh OpenY installation | http://ci.openymca.org/build1316 |
| Upgraded(upgrade path) installation | http://upgrade.openy-dev.ffwua.com/build1316 |
| Installation process | http://install.openy-dev.ffwua.com/build1316/install.php |

--------------
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.openymca.org:8080/job/PR_BUILDER/1316/

**Comment by gundevel** [comment]
Build comment file: 
--------------
===============
 
| Environment | Link |
| ------------- | ------------- |
| Fresh OpenY installation | http://ci.openymca.org/build1327 |
| Upgraded(upgrade path) installation | http://upgrade.openy-dev.ffwua.com/build1327 |
| Installation process | http://install.openy-dev.ffwua.com/build1327/install.php |

--------------
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.openymca.org:8080/job/PR_BUILDER/1327/

**Review by ddrozdik — APPROVED** [review]

**Comment by gundevel** [comment]
Behat test results file URL: http://ci.openymca.org/build1327/profiles/contrib/openy/build/reports/behat

File: themes/openy_themes/openy_rose/templates/paragraph/paragraph--small-banner.html.twig [hunk 1]
```diff
@@ -60,6 +60,11 @@
           <h3 class="banner-title">
             {{ content.field_prgf_headline }}
           </h3>
+          {% if content.field_prgf_description.0['#text'] is not empty %}
```
Comment (alexschedrov) [inline themes/openy_themes/openy_rose/templates/paragraph/paragraph--small-banner.html.twig]: We should set variable like here https://github.com/ymcatwincities/openy/pull/393/files#diff-099c1735c766cb3803f7f2f2bca786d4R48. We should never use array properties in twig templates. If we need to do this, we should define new variables above and then use them in conditions.

**Review by alexschedrov — CHANGES_REQUESTED** [review]

**Comment by gundevel** [comment]
Build comment file: 
--------------
===============
 
| Environment | Link |
| ------------- | ------------- |
| Fresh OpenY installation | http://ci.openymca.org/build1328 |
| Upgraded(upgrade path) installation | http://upgrade.openy-dev.ffwua.com/build1328 |
| Installation process | http://install.openy-dev.ffwua.com/build1328/install.php |

--------------
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.openymca.org:8080/job/PR_BUILDER/1328/

**Comment by gundevel** [comment]
Behat test results file URL: http://ci.openymca.org/build1328/profiles/contrib/openy/build/reports/behat

**Comment by Sanchiz** [comment]
retest this please

File: themes/openy_themes/openy_rose/templates/paragraph/paragraph--small-banner.html.twig [hunk 2]
```diff
@@ -46,6 +46,7 @@
   ]
 %}
 {% set color = '#' ~ content.field_prgf_color['#items'].entity.field_color.value %}
+{% set description_text = content.field_prgf_description.0['#text'] %}
```
Comment (alexschedrov) [inline themes/openy_themes/openy_rose/templates/paragraph/paragraph--small-banner.html.twig]: That's better, however `content.field_prgf_description.0['#text']` may be empty from the begining.

We should add condition here, when we setup description text value. `description_text` should be just like a flag, `true` or `false`.

**Review by alexschedrov — CHANGES_REQUESTED** [review]

**Comment by gundevel** [comment]
Build comment file: 
--------------
===============
 
| Environment | Link |
| ------------- | ------------- |
| Fresh OpenY installation | http://ci.openymca.org/build1335 |
| Upgraded(upgrade path) installation | http://upgrade.openy-dev.ffwua.com/build1335 |
| Installation process | http://install.openy-dev.ffwua.com/build1335/install.php |

--------------
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.openymca.org:8080/job/PR_BUILDER/1335/

**Comment by gundevel** [comment]
Behat test results file URL: http://ci.openymca.org/build1335/profiles/contrib/openy/build/reports/behat

**Comment by gundevel** [comment]
Build comment file: 
--------------
===============
 
| Environment | Link |
| ------------- | ------------- |
| Fresh OpenY installation | http://ci.openymca.org/build1342 |
| Upgraded(upgrade path) installation | http://upgrade.openy-dev.ffwua.com/build1342 |
| Installation process | http://install.openy-dev.ffwua.com/build1342/install.php |

--------------
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.openymca.org:8080/job/PR_BUILDER/1342/

**Comment by gundevel** [comment]
Behat test results file URL: http://ci.openymca.org/build1342/profiles/contrib/openy/build/reports/behat

**Comment by Sanchiz** [comment]
retest this please

**Comment by bjwilkins** [comment]
Build comment file: 
--------------
===============
 
| Environment | Link |
| ------------- | ------------- |
| Fresh OpenY installation | http://ci.openymca.org/build1345 |
| Upgraded(upgrade path) installation | http://upgrade.openy-dev.ffwua.com/build1345 |
| Installation process | http://install.openy-dev.ffwua.com/build1345/install.php |

--------------
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.openymca.org:8080/job/PR_BUILDER/1345/

**Comment by bjwilkins** [comment]
Behat test results file URL: http://ci.openymca.org/build1345/profiles/contrib/openy/build/reports/behat

**Review by alexschedrov — APPROVED** [review]
Approved.

**Pull request merged**